scope/inscope.txt

```diff
@@ -144,9 +144,11 @@
 
 # Global IP Action AG
 *.globalipaction.ch
+*.glipac.ch
 
```

Comment:
You could also add *.scsstatic.ch

Reply:
Defunct? Examples: 
https://sdx.scsstatic.ch/
https://rcp.scsstatic.ch/content/dam/assets/b2c/promotionen/apple-iphone-cropped.png

Reply:
you are right. will add them to the scope, many thanks!